static/app/test/specs/entities.spec.js

```diff
@@ -8,27 +8,102 @@ var Radio = require('backbone.radio');
-            it('unregisters entities:moves request handler');
-            it('unregisters entities:move request handler');
-            it('unregisters entities:move:reset request handler');
+            it('unregisters entities:spots request handler', function(){
+                var entitiesAPI = new EntitiesAPI();
+                entitiesAPI.destroy();
+                var result = channel.request('entities:spots');
+                assert.isUndefined(result, 'entities:spots returns nothing');
+            });
+            it('unregisters entities:moves request handler', function(){
+                var entitiesAPI = new EntitiesAPI();
+                entitiesAPI.destroy();
+                var result = channel.request('entities:moves');
+                assert.isUndefined(result, 'entities:moves returns nothing');
+            });
+            it('unregisters entities:move request handler', function(){
+                var entitiesAPI = new EntitiesAPI();
+                entitiesAPI.destroy();
+                var result = channel.request('entities:move');
+                assert.isUndefined(result, 'entities:move returns nothing');
+            });
+            it('unregisters entities:move:reset request handler', function(){
+                var entitiesAPI = new EntitiesAPI();
+                entitiesAPI.destroy();
+                var result = channel.request('entities:move:reset');
+                assert.isUndefined(result, 'entitites:move:reset');
+            });
         });
 
         describe('resetMove', function(){
-            it('destroys the current moves and passes options to move.destroy');
-            it('resets move with a new move');
+            it('destroys the current moves and passes options to move.destroy', function(){
+                var entitiesAPI = new EntitiesAPI();                
+                var move = channel.request('entities:move');
+                var options = {something: "something"};
+                var moveDestroySpy = sinon.spy(move, 'destroy');
+
+                entitiesAPI.resetMove(options);
+                assert.isTrue(moveDestroySpy.called, 'resetMove calls move.destroy');
+                assert.isTrue(moveDestroySpy.calledWith(options), 'resetMove passes options to move.destroy');
+            });
+            it('resets move with a new move', function(){
+                var entitiesAPI = new EntitiesAPI();
+                var move = channel.request('entities:move');
+                this.resetMoveStub = sinon.stub(EntitiesAPI.prototype, 'resetMove', function(){
+                    move.destroy();
+                    return new Move({ user: move.get('user') });
+                });
+                
+                var result = entitiesAPI.resetMove();
+                assert.notEqual(move, result, 'resetMove resets with a new move');
+                this.resetMoveStub.restore();
+            });
         });
     });
-});
+});
```

Comment:
[Git likes every file to end in a new line](http://stackoverflow.com/questions/5813311/no-newline-at-end-of-file) and Github displays this icon when there's no new line. So add a newline!